tpgtools/main.go

```diff
@@ -35,6 +35,7 @@ var fPath = flag.String("path", "", "path to the root service directory holding
 var tPath = flag.String("overrides", "", "path to the root directory holding overrides files")
 var cPath = flag.String("handwritten", "handwritten", "path to the root directory holding handwritten files to copy")
 var oPath = flag.String("output", "", "path to output generated files to")
+var sPath = flag.String("samples", "samples", "path to the directory holding the samples and substitution data")
```

Comment:
non-related:
This is consistent with `fpath` and `tpath` but I wonder why we offer the ability to override the paths. Do we really expect anyone to give a non-fixed path?

Reply:
going to be removing this anyways in the future when I migrate to samples.. leaving for now